atomix/cluster/src/main/java/io/atomix/raft/partition/impl/RaftNamespaces.java

```diff
@@ -83,9 +77,7 @@ public final class RaftNamespaces {
           .register(MemberId.class)
           .register(RaftMember.Type.class)
           .register(Instant.class)
-          .register(Configuration.class)
-          .register(ApplicationEntry.class)
-          .register(RaftLogEntry.class)
+          .nextId(528) // for backwards compatibility
```

Comment:
:upside_down_face: TIL
:question: Why? and do we no longer need these classes?

Reply:
We do use them internally, but not as part of our protocol. Meaning you can change them as long as the SBE serialization handles them properly, but you don't have to worry about messing up Kryo serialization which is based on the structure of your class (e.g. field ordering and such).

Since changing classes that need to be serialized with Kryo is, uh, error prone (or at least you have to be very careful), I wanted to make it clear which you can and which you cannot change.

Reply:
Thanks :+1:

Reply:
I guess we can use this trick on other places where I used Void.class :D

Reply:
Oh, I forgot about that, that's also clever and would've been less effort than what I did (manually counting everything :sweat_smile:)